subsys/logging/log_backend_rtt.c

```diff
@@ -16,12 +16,18 @@
 
 #define DROP_MSG "\nmessages dropped:    \r"
 #define DROP_MSG_LEN (sizeof(DROP_MSG) - 1)
-
+#define MESSAGE_SIZE CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE
+#define CHAR_BUF_SIZE 1
+#define RETRY_DELAY_MS 10 /* Long enough to detect host presence */
```

Comment:
Doesn't we have Kconfig options for that?
(`LOG_BACKEND_RTT_RETRY_CNT` and `LOG_BACKEND_RTT_RETRY_DELAY_MS`)

Reply:
those options are present only in blocking mode. In drop mode, in normal operation they are not used because data is dropped when no room (that covers disconnected host case). In drop mode, it will only do blocking write in panic and those settings are for this case. I didn't want to spoil kconfig. If user picks drop mode then it's confusing to set also retry timeout.